card/lib/card/name.rb

```diff
@@ -41,5 +41,17 @@ def trait tag_code
     def code
       Card::Codename[ Card.fetch_id self ]
     end
+
+
+    def is_a_field_of? context_name
```

Comment:
Good point. Checking whether the relative name starts with a '+'  seems like the right solution for all cases.

( I had to add the additional case in line 53 is for structured cardtypes. The page /new/mycardtype has an empty name so the absolute name of a field is the same as the relative name. Your suggestion would handle that)